modules/ROOT/pages/bat-reporting-task.adoc

```diff
@@ -233,4 +234,20 @@ And the result is:
 {
   "result": "The suite <Hello world suite> with assertion <answer 200> is ok"
 }
-----
\ No newline at end of file
+----
+
+== Email
+
+When using the Anypoint Platform you can create an email reporter to enable email alerts to several accounts.
+
+_This option is only available in the Anypoint Platform since it uses the Anypoint Platform Alert Notification System._
```

Comment:
How does this system need to be configured?

Reply:
@rddheath I changed the doc. can you take a look?